kolibri/core/assets/src/views/CoreBase/ScrollingHeader.vue

```diff
@@ -46,6 +46,10 @@
       };
     },
     computed: {
+      scrollPositionOr0() {
```

Comment:
Make this `scrollPositionOrZero` or something like `positiveScrollPositon` so it's easier to distinguish `0` and `O` for people with fonts that don't distinguish them.